data_provenance/to-dolma.py

```diff
@@ -66,6 +66,22 @@
 SOURCE_NAME = "Data Provenance Initiative"
 
 
+def get_unique_ids(folder_path):
```

Comment:
Probably add a comment that this assumes that there aren't subdirectories, or use a glob that supports finding nested jsonl.gz files.

Reply:
Added glob support and rerunning now

Reply:
Reran and works